components/clubs-page.jsx

```diff
@@ -208,7 +166,29 @@ var ClubsPage = React.createClass({
           <Quote/>
         </section>
         <section>
-          <IconLinks/>
+          <IconLinks links={[
+            {
+              linkTo: "fixme",
+              imgSrc: "/img/icon-curriculum.svg",
+              imgAlt: "icon curriculum",
+              head: "Curriculum",
+              subhead: "Modular Web Literacy curriculum"
+            },
+            {
+              linkTo: "fixme",
+              imgSrc: "/img/icon-connect.svg",
+              imgAlt: "icon connect",
+              head: "Connect",
+              subhead: "Connect with other Club Leaders"
+            },
+            {
+              linkTo: "fixme",
+              imgSrc: "/img/icon-tips.svg",
+              imgAlt: "icon tips",
+              head: "Helpful Tips",
+              subhead: "Tips for running your Club"
+            }
+          ]}/>
```

Comment:
Oh, another advantage of this approach is that I think it would obviate the need to put a `key` prop on every `IconLink`. Which makes sense, since those links are so static.